admins/pageflow/entry.rb

```diff
@@ -33,7 +33,10 @@ module Pageflow
     sidebar :folders, :only => :index do
       text_node(link_to(I18n.t('pageflow.admin.entries.add_folder'), new_admin_folder_path, :class => 'new'))
       grouped_folder_list(Folder.accessible_by(Ability.new(current_user), :read),
-                          :class => authorized?(:manage, Folder) ? 'editable' : nil,
+                          class: if params[:id] &&
+                                    authorized?(:configure_folder_on, resource.account)
+                                   'editable'
+                                 end,
```

Comment:
The grouped folder list is only every displayed on the index page. `params[:id]` will never be present. `editable` will have to be decided inside the grouped folder list for each account separately.